### PR TITLE
fixes fr #1440

### LINKED ISSUE #1440: All mentions of wiki.developer.mozilla.org should be removed
E.g. https://github.com/mdn/translated-content/blob/08a3b863c23142aa3b1b6fa565e5346cb6d74a99/files/fr/mozilla/add-ons/webextensions/api/userscripts/register/index.html#L40
or https://github.com/mdn/translated-content/blob/08a3b863c23142aa3b1b6fa565e5346cb6d74a99/files/ja/learn/forms/index.html#L47

If you see something like:
```html
<a href="https://wiki.developer.mozilla.org/ja/docs/Learn/Forms/Other_form_controls">その他のフォームコントロール</a>
```
change it to:
```html
<a href="/ja/docs/Learn/Forms/Other_form_controls">その他のフォームコントロール</a>
```

### PR DESCRIPTION
**Pull request by SphinxKnight**
This intends to fix #1440 for French docs.
Once this will have been merged, I intend to get rid of absolute URL containing developer.mozilla.org as well.

### CODE REVIEW

File: files/fr/learn/css/building_blocks/backgrounds_and_borders/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
...
    <th scope="row">Prérequis:</th>
-   <td>Compétences informatique basiques , <a href="https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/Installing_basic_software">logicels de base installés</a>, connaissance simple en <a href="https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/Dealing_with_files">manipulation de fichiers</a>, les bases du HTML (voir <a href="/en-US/docs/Learn/HTML/Introduction_to_HTML">Introduction au HTML</a>), et une esquisse du fonctionnement du CSS (voir <a href="https://wiki.developer.mozilla.org/en-US/docs/Learn/CSS/First_steps">premiers pas en CSS</a>. )</td>
+   <td>Compétences informatique basiques , <a href="https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/Installing_basic_software">logicels de base installés</a>, connaissance simple en <a href="https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/Dealing_with_files">manipulation de fichiers</a>, les bases du HTML (voir <a href="/en-US/docs/Learn/HTML/Introduction_to_HTML">Introduction au HTML</a>), et une esquisse du fonctionnement du CSS (voir <a href="/fr/docs/Learn/CSS/First_steps">premiers pas en CSS</a>. )</td>
```
Comment (peterbe) [inline files/fr/learn/css/building_blocks/backgrounds_and_borders/index.html:15]:
```suggestion
   <td>Compétences informatique basiques , <a href="https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/Installing_basic_software">logicels de base installés</a>, connaissance simple en <a href="/fr/Learn/Getting_started_with_the_web/Dealing_with_files">manipulation de fichiers</a>, les bases du HTML (voir <a href="/fr/docs/Learn/HTML/Introduction_to_HTML">Introduction au HTML</a>), et une esquisse du fonctionnement du CSS (voir <a href="/fr/docs/Learn/CSS/First_steps">premiers pas en CSS</a>. )</td>
```

File: files/fr/learn/server-side/django/development_environment/index.html [hunk 2]
```diff
@@ -48,7 +48,7 @@ <h3 id="Quelles_sont_les_options_dinstallation_de_Django">Quelles sont les optio
 <p>Chacune de ces options requiert une configuration et une installation légèrement différente. Les sous-sections ci-dessous vous expliquent différents choix. Dans le reste de l'article, nous vous montrerons comment installer Django sur un nombre restreint de systèmes d'exploitation, et nous supposerons que cette installation aura été suivie pour tout le reste du module.</p>
 
 <div class="note">
-<p><strong>Note</strong>: D'autres options d'installation possibles sont traitées dans la documentation officielle de Django. Les liens vers la <a href="https://wiki.developer.mozilla.org/en-US/docs/Learn/Server-side/Django/development_environment$translate?tolocale=fr#">documentation appropriée peuvent-être trouvés ci-dessous</a>.</p>
+<p><strong>Note</strong>: D'autres options d'installation possibles sont traitées dans la documentation officielle de Django. Les liens vers la <a href="/fr/docs/Learn/Server-side/Django/development_environment$translate?tolocale=fr#">documentation appropriée peuvent-être trouvés ci-dessous</a>.</p>
```
Comment (peterbe) [inline files/fr/learn/server-side/django/development_environment/index.html:51]:
```suggestion
<p><strong>Note</strong>: D'autres options d'installation possibles sont traitées dans la documentation officielle de Django. Les liens vers la <a href="/fr/docs/Learn/Server-side/Django/development_environment">documentation appropriée peuvent-être trouvés ci-dessous</a>.</p>
```

**Review by peterbe — CHANGES_REQUESTED** [review]
If you can heed my two suggestions you don't need to come back to get rid of the `https://developer.mozilla.org` part later.

File: files/fr/learn/css/building_blocks/backgrounds_and_borders/index.html [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
 slug: Learn/CSS/Building_blocks/Backgrounds_and_borders
+tags:
+  - Background
+  - Beginner
+  - CSS
+  - Image
+  - Learn
+  - Position
+  - borders
+  - color
```
Comment (peterbe) [inline files/fr/learn/css/building_blocks/backgrounds_and_borders/index.html:12]: Tags are never used or read in the translated content. No point adding them.

**Review by peterbe — APPROVED** [review]
I'm hoping someone else will do a final review but the things I mentioned initially seem to be resolved. Thanks!

File: files/fr/learn/css/building_blocks/backgrounds_and_borders/index.html (hunk 3 above)
Comment (tristantheb) [reply files/fr/learn/css/building_blocks/backgrounds_and_borders/index.html:12]: Ok, I finish to check all pages and I remove the tags of this page ^^

**Review by tristantheb — APPROVED** [review]
Cleanup complete @SphinxKnight

**Pull request merged**